features/testbot/matching_turns.feature

```diff
@@ -17,21 +17,13 @@ Feature: Turn directions/codes
         And the ways
             | nodes |
             | xa    |
-            | xb    |
             | xc    |
-            | xd    |
             | xe    |
-            | xf    |
             | xg    |
-            | xh    |
             | xi    |
-            | xj    |
             | xk    |
-            | xl    |
             | xm    |
-            | xn    |
             | xo    |
-            | xp    |
 
```

Comment:
Technically, this makes the test a lot simpler, but its only designated to check for turns, not the matching.
We have to deal with a single node of degree 16, though. This makes it impossible to handle the instructions reasonably with only 8 instructions. Maintaining this, to handle more angles than these 8 seems unreasonable.